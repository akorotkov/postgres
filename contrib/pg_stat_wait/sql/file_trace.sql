select pg_start_trace(0, '/tmp/pg_stat_wait.trace');
select pg_is_in_trace(0);
select pg_stop_trace(0);

