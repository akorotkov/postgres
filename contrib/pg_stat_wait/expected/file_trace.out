select pg_start_trace(0, '/tmp/pg_stat_wait.trace');
 pg_start_trace 
----------------
 
(1 row)

select pg_is_in_trace(0);
 pg_is_in_trace 
----------------
 t
(1 row)

select pg_stop_trace(0);
 pg_stop_trace 
---------------
 
(1 row)

